### DISCUSSION TITLE
Add timestamp to Documentation

### DISCUSSION THREAD

**Discussion by ckartik** [MinaProtocol/mina#12648] [Ideas]
Kind of a nit ask, but would it be possible to add a timestamp to some of the articles and tutorials that are created in the documentation? 
Sometimes new things pop into sections like these: https://docs.minaprotocol.com/zkapps/advanced-snarkyjs/events and I'm curious when they were added/updated! 😄

**Comment by johnathon-b** [comment]
You can select "Edit this page" on the top left of the article to see the git repo and dates the article content was edited or added.
